test/CMakeLists.txt

```diff
@@ -238,6 +239,7 @@ make_test(SRC medfilt.cpp)
 make_test(SRC median.cpp)
 make_test(SRC memory.cpp)
 make_test(SRC memory_lock.cpp)
+make_test(SRC memorymanager.cpp)
```

Comment:
We already have a file `memory.cpp` for memory manager tests. If the new tests are not radically different from the ones in memory.cpp, we should move the contents of this file to `memory.cpp`